### ISSUE TITLE
stopped working with v1.54

### ISSUE THREAD

**Issue by ctf0** [trentrand/vscode-git-rebase#10]
```
Version: 1.54.0-insider
Commit: cd97629c4458c05b16a1fd444d21c61295ba2529
Date: 2021-02-09T08:34:46.198Z
Electron: 11.2.3
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 19.6.0
```

```
Running the contributed command: 'extension.setRebaseActionFixup' failed.
```

**Comment by trentrand** [comment]
Hey @ctf0! Thank you very much for reporting this issue, it seems that the extension was using private properties from the API that were removed in version 1.54.0-insider.

I've pushed a fix and tagged it with `1.0.6`. I'll follow-up on issue once it's published on the VSCode Extension Marketplace.

**Comment by trentrand** [comment]
I published a fix in [trentrand.git-rebase-shortcuts@1.0.6](https://marketplace.visualstudio.com/items?itemName=trentrand.git-rebase-shortcuts) to the VisualStudio Extension Marketplace!

I'm going to close this issue as fixed, but please leave a comment once you give it a try and let me know if you run into any further issues!
